src/ui/dashboard.js

```diff
@@ -228,18 +230,25 @@ export class DashboardScene extends Phaser.Scene {
             .image(592, 168, "quests-button")
             .setOrigin(0, 0)
             .setDepth(2)
-            .setInteractive();
-
-        // Quest text
-        // TODO: show all quests
-        this.killQuestText = this.add.text(555, 256, "", { fill: "white" }).setDepth(3);
+            .setInteractive()
+            .on("pointerdown", () => {
+                this.showQuests(true);
+            });
 
-        this.quests.button.on("pointerdown", () => {
-            this.showQuests(true);
+        // Add scrollable window for quests
+        this.quests.scrollWindow = new ScrollWindow({
+            name: "quests",
+            x: 541,
+            y: 250,
+            width: 175,
+            height: 213,
+            numColumns: 1,
+            padding: 10,
         });
+        this.scene.add(this.quests.scrollWindow.name, this.quests.scrollWindow, true);
+        this.quests.scrollWindow.refresh();
 
-        // Set and hide quests on startup
-        this.updateKillQuestText();
+        this.quests.list = new QuestList(this, this.quests.scrollWindow);
```

Comment:
Try initializing the scroll window in QuestList instead of passing it in. If it works and looks nice we should do it for Clan as well

Reply:
Okay I can see how it’s useful there. I guess for multiple prerequisites we would change it to a list and loop through them?

Reply:
I've done the deed but the scrollbars don't seem to be working... Just tested them on master and they don't seem to be working, so I guess something slipped through.

Reply:
> Okay I can see how it’s useful there. I guess for multiple prerequisites we would change it to a list and loop through them?

Yeah most likely we will need to handle all levels as having the potential for multiple prereqs and loop though them

Reply:
> I've done the deed but the scrollbars don't seem to be working... Just tested them on master and they don't seem to be working, so I guess something slipped through.

Chat log and shop seem to be working well. I'm not sure why the clan window isn't; this scroll bar has given me so much pain lol. When you have a chance you can try your own rendition of it, I feel like I've rewritten it 3 times now